src/nft/pages/collection/Components.tsx

```diff
@@ -0,0 +1,29 @@
+import { Box } from 'nft/components/Box'
+import { Row } from 'nft/components/Flex'
+import * as styles from 'nft/pages/collection/common.css'
+
+interface ActivitySwitcherProps {
+  showActivity: boolean
+  toggleActivity: () => void
+}
+
+export const ActivitySwitcher = ({ showActivity, toggleActivity }: ActivitySwitcherProps) => {
```

Comment:
This file should be named `ActivitySwitcher`.

As mentioned in an earlier comment, the definitions for showActivity and toggleActivity should be adjacent for clarity. Another option for that is to make this an uncontrolled component, and have it define those things itself.

Reply:
removing activity until its implemented later